src/moparser.js

```diff
@@ -135,7 +135,7 @@ Parser.prototype._handleCharset = function (headers) {
  * Adds a translation to the translation object
  *
  * @param {String} msgid Original string
- * @params {String} msgstr Translation for the original string
+ * @param {String} msgstr Translation for the original string
```

Comment:
Good catch. Also pre-TS PR please. You get the idea.

Reply:
OK let's do this: I remove everything that is not strictly necessary for this PR including the fixes for deprecated commands (substr) and the fixes for types (such as params)

I follow this PR with dedicated PRs fixing deprecations and types and again follow this pr again, another one, where I implement jsDoc with strict type checking. 

Typescript is a decision I leave completely up to you (for me it would be to do), but what we are doing will make this step incredibly easy if required

Reply:
but they must be fixed otherwise the problem is this:

![typed](https://github.com/smhg/gettext-parser/assets/8550908/4e38d859-b171-4c76-9b67-57593649649b)